client.go

```diff
@@ -168,6 +170,8 @@ type marathonClient struct {
 //		config:			the configuration to use
 func NewClient(config Config) (Marathon, error) {
 	// step: if no http client, set to default
+
+	fmt.Println(config.URL)
```

Comment:
In general, logging should happen through the [log writer](https://github.com/gambol99/go-marathon/blob/697a616b3996092969b86950b1e555e3b4391945/config.go#L45). I don't really see a need to log the URL though. Could you please explain why you think it's necessary?